ed/idlpatches/web-animations-2.idl.patch

```diff
@@ -1,31 +1,21 @@
-From c23c657bba54488f8223ef05634aa1a12bd172bb Mon Sep 17 00:00:00 2001
-From: Francois Daoust <fd@tidoust.net>
-Date: Thu, 24 Jun 2021 10:55:15 +0200
-Subject: [PATCH] Drop duplicate `fillMode` enum, fix `ComputedEffectTiming`
+From ea2e5beb27b21ea54c3633c0282e71f1d9e14393 Mon Sep 17 00:00:00 2001
+From: Kagami Sascha Rosylight <saschanaz@outlook.com>
+Date: Thu, 1 Jul 2021 13:35:40 +0200
+Subject: [PATCH] Drop duplicate `fillMode` enum definition
 
 The spec is a delta spec and re-defines the enum to change the meaning of one of
-its value. It also extends but also modifies the definition of the
-`ComputedEffectTiming` dictionary.
+its values.
 
 Patch will likely need to be kept around for as long as the spec remains a delta
 spec.
 ---
- ed/idl/web-animations-2.idl | 4 +---
- 1 file changed, 1 insertion(+), 3 deletions(-)
+ ed/idl/web-animations-2.idl | 2 --
+ 1 file changed, 2 deletions(-)
 
 diff --git a/ed/idl/web-animations-2.idl b/ed/idl/web-animations-2.idl
-index 1b5db8b13..c03be662d 100644
+index 1b5db8b13..2bc7aa412 100644
 --- a/ed/idl/web-animations-2.idl
 +++ b/ed/idl/web-animations-2.idl
-@@ -30,7 +30,7 @@ partial dictionary OptionalEffectTiming {
-     double playbackRate;
- };
- 
--partial dictionary ComputedEffectTiming {
-+dictionary ComputedEffectTiming {
-     CSSNumberish         startTime;
-     CSSNumberish         endTime;
-     CSSNumberish         activeDuration;
 @@ -38,8 +38,6 @@ partial dictionary ComputedEffectTiming {
      CSSNumberish?        progress;
```

Comment:
we still need the other fields here?

Reply:
Not sure I follow? This stops patching ComputedEffectTiming in level 2 and instead do it in level 1.

Reply:
I may easily be confused, since reviewing diff of patches is a bit hard to process for me; but my impression is that the two patches would end up with a consolidated `ComputerEffectTiming` dictionary without `startTime`, and with the wrong type on `endTime` and `activeDuration`

Reply:
Since my patch still removes all fields from level 1 spec including `startTime`, `endTime` and `activeDuration` but not `currentIteration`, they should follow level 2 definition, which is consistent to the current situation (unless I'm confused by something).

Reply:
ah, I see - thanks, I was indeed confused by the multiple mental patching.